src/api/engine/engineApi.ts

```diff
@@ -61,4 +62,18 @@ export class EngineApi extends Base {
 
     return result.status;
   }
+
+  async suspendEngine(name: string) {
+    await this.patch<EmptyResponse>(`${ENDPOINT}/${name}`, {
+      body: { suspend: true },
+    });
+    return;
+  }
+
+  async resumeEngine(name: string) {
+    await this.patch<EmptyResponse>(`${ENDPOINT}/${name}`, {
+      body: { suspend: true },
```

Comment:
`{ suspend: true }` shouldn't it be false? the resume call looks the same as the suspend, is this really so?

Reply:
Ah, indeed, that's bad :D